sdk/core/Azure.Core/tests/public/ModelReaderWriterTests/Models/DogListProperty.cs

```diff
@@ -41,10 +41,11 @@ public DogListProperty()
             return DeserializeDogListProperty(jsonDocument.RootElement, ModelReaderWriterOptions.DefaultWireOptions);
         }
 
-        public static implicit operator RequestContent(DogListProperty dog)
-        {
-            return RequestContent.Create(dog, ModelReaderWriterOptions.DefaultWireOptions);
-        }
+        // TODO: Move this to Experimental until we GA the API
```

Comment:
We'd rather not add a dependency on Azure.Core.Experimental to Azure.Core.Public.Tests project, but I think we should be able to factor the test projects so this isn't needed if we decide to move forward with this approach.